test/parallel/test-http-missing-header-separator-cr.js

```diff
@@ -26,7 +26,7 @@ server.listen(0, common.mustSucceed(() => {
   let response = '';
 
   client.on('data', common.mustCall((chunk) => {
-    response += chunk.toString('utf-8');
+    response += chunk;
   }));
 
   client.setEncoding('utf8');
```

Comment:
Fwiw, my preferred way is `client.setEncoding('utf8').on('data', ...')` :smile:

Ultimately, it doesn't matter. If we do clean this up, it might also be a good idea to do so across all tests for consistency.